packages/material-ui/src/Chip/Chip.js

```diff
@@ -8,7 +8,6 @@ import useForkRef from '../utils/useForkRef';
 import unsupportedProp from '../utils/unsupportedProp';
 import capitalize from '../utils/capitalize';
 import ButtonBase from '../ButtonBase';
-import '../Avatar'; // So we don't have any override priority issue.
```

Comment:
Do you think a "harmless" side-effect such as `Avatar.propTypes = Avatar.propTypes` (or similar, like `Avatar.a = Avatar.a`) might suffice? I'm on holiday rn so I don't have much time to look into this issue myself, sorry.

Reply:
After more reflexion, I think that having to pull out another module to get the correct side effect isn't right. It prevents tree-shaking. Autocomplete users shouldn't bundle the Avatar if they don't use it.

Reply:
Enjoy your holidays 👍